dashboard.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import avahi
+
+from flask import Flask, request, render_template, make_response, current_app, json, Response, jsonify, redirect, url_for
+
+app = Flask(__name__)
+
+def get_device_list():
+	'''Returns a list of ips of devices'''
+	device_list = []
+	avahi.run()
```

Comment:
Overall, this commit is awesome, but I think we might want to refactor this bit.

This is running the avahi browser on every homepage request, which is a bit inefficient? Ideally I think we'd run the Avahi call from Cron every minute or two, and then read the BLINKT_DEVICES file created by it, in order to get _recent-ish_ status.

Reply:
Good idea, okay. You can delete that line and then add avahi.py to the setting up cron instructions in the readme then